### DISCUSSION TITLE
command not found: podman

### DISCUSSION THREAD

**Discussion by MrAnyx** [containers/podman#21913] [Q&A]
Hello,

I just started using Podman yesterday, so I'm a bit of a novice. I really like the fact that it's lightweight but I'm missing a feature that Docker has : being able to use the docker cli within the WSL2 Ubuntu distribution.

I installed Podman-desktop 1.7.1 on windows 10 and I created the default podman machine to manage pods and containers.

> I didn't install podman on WSL2 Ubuntu using `apt`, I installed it on my Windows operating system using the podman-desktop app.

I have the same problem with podman as with Docker. The fact that the container is really slow because of the file system between windows and WSL is very frustrating. So I moved my project to the WSL2 machine to make it faster, but now I can't access the podman client.

Am I missing something?

**Comment by baude** [comment]
this is sort of the dilemma people are facing with all these various solutions.  Most people just download the windows client and use the default podman machine experince.  But as you point out, you can also run it in a WSL instance without the windows client.  I think there is even a third hybrid version of this experience too.  you should be able to use the podman inside the wl podman machine as well.  i dont have windows at my fingertips rn but that should work

**Reply by MrAnyx** [reply]
Do you mean that I can install podman as a normal library on my Ubuntu WSL 2 distro using `apt-get` ?

**Reply by MrAnyx** [reply]
Is it better to create my project in my Ubuntu wsl2 disto or directly in the podman-default machine ?

**Reply by baude** [reply]
yeah, i was saying you *could* do that.  Keep in mind, generally ubuntu, depending on your choice of which ubuntu, can be fairly far behind what is available in terms of releases.  if you want a drop dead simple user experience, then podman machine is a great approach; if you are looking to alter that experience in a way that does not make sense to contribute back, then maybe rolling your own makes sense.  There is no better; only what you want to do.
